edumanage/migrations/0011_edb21_server_schema.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2022-10-19 13:43
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('edumanage', '0010_auto_bexact_noop'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='RealmServer',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('server_name', models.CharField(help_text='IP address | FQDN hostname', max_length=80)),
+                ('server_type', models.PositiveIntegerField(choices=[(0, 'UDP'), (1, 'TLS'), (2, 'F-ticks')], help_text='Realm server type')),
+                ('realm', models.ForeignKey(blank=True, on_delete=django.db.models.deletion.CASCADE, to='edumanage.Realm')),
```

Comment:
```suggestion
                ('realm', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='servers', to='edumanage.Realm')),
```

The migration might have been generated from a different version of the model?  This change would align it with the `realm` definition in `edumanage/models.py`  .... and would stop Django from trying to create an additional migration to resolve the difference.

Reply:
It is entirely possible. The code was done a while ago, and there have been a number of changes since. But also we have our own branch with several local patches, and it is possible that I started this from the wrong point. Happy that this has more eyes to pick this up.